activesupport/lib/active_support/testing/strict_warnings.rb

```diff
@@ -12,7 +12,16 @@ module RaiseWarnings
     /Ignoring .*\.yml because it has expired/,
     /Failed to validate the schema cache because/,
   )
+
+  SUPPRESSED_WARNINGS = Regexp.union(
+    # TODO: remove if https://github.com/mikel/mail/pull/1557 or similar fix
+    %r{/lib/mail/parsers/.*statement not reached},
```

Comment:
Hum, doesn't seem to work, maybe it's loaded too late? https://buildkite.com/rails/rails/builds/94131#01868059-d70b-47ae-8cd2-3b6a1fa78824/1035-1038

Reply:
Hum, right it seem to fix actionmailer, but not actionmailbox.

Reply:
[ba028a4](https://github.com/rails/rails/pull/47484/commits/ba028a48f6d1febf01a74058b753b340188eb6ef) should fix it, I can squash if you think they should go together

Reply:
Nah it's ok